cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1261,6 +1298,22 @@ describe('CourseOutlinePage', function() {
             expect($('.edit-show-correctness').length).toBe(1);
         });
 
+        it('can show correct editors for self_paced course with custom pacing', function (){
```

Comment:
I wonder if all of these new tests should live inside their own `describe` for "custom pacing". What do you think?

Reply:
hmm yeah that sounds like a good idea! Currently all of the tests are under description "Subsection" - would it be cleaner to have a nested `describe` within "Subsection" tests or a separate `describe` outside?